### PR TITLE
Text file support for TabPort Extension

### PR DESCRIPTION
**Pull request by F1nn-T**

### CODE REVIEW

File: src/__tests__/Utils/FileFormat.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { DeterminUrlType, FileFormats } from "../../utils/FileFormat";
```
Comment (Jitensid) [inline src/__tests__/Utils/FileFormat.test.js]: Hi. it should be **determineUrlType** a small typo
Also, keep variable names as camelCase 
Ex: **FileFormats** should be **fileFormats** and so on

**Comment by Jitensid** [comment]
Also can you pull again from the main branch so that you have the latest workflow config because currently not all checks are running in this Pull Request.

File: src/__tests__/Utils/FileFormat.test.js (hunk 1 above)
Comment (F1nn-T) [reply src/__tests__/Utils/FileFormat.test.js]: I fixed the typo.

**Comment by F1nn-T** [comment]
I fixed the typo and synced the fork, can you check again @Jitensid .

**Comment by Jitensid** [comment]
Thanks @F1nn-T

**Pull request merged**